doc/getting-started/rt_industry.rst

```diff
@@ -224,49 +224,33 @@ Use the ACRN industry out-of-the-box image
 Install and launch the Preempt-RT VM
 ************************************
 
+In this section, we will use :ref:`virtio-blk` to launch the Preempt-RT VM.
+If you need a better performance, follow the :ref:`build-the-service-vm-kernel`
+to replace ACRN kernel on the Service VM, then :ref:`passthrough SATA disk <passthru rtvm>`
+to launch Preempt-RT VM.
+
 #. Log in to the Service VM with root privileges.
 
 #. Download the Preempt-RT VM image::
 
-   # wget https://github.com/projectacrn/acrn-hypervisor/releases/download/acrn-2020w01.1-140000p/preempt-rt-32030.img.xz
+   # wget https://github.com/projectacrn/acrn-hypervisor/releases/download/acrn-2020w12.5-140000p/preempt-rt-32680.img.xz
```

Comment:
Do we describe somewhere how one can build such image himself?

Reply:
I found this link: <https://github.com/projectacrn/acrn-hypervisor/releases/latest>
Is it better ?

Reply:
No, the link is fine. I was curious to see if we had more details on what is included in the image. How is it different from a standard image file (normal User VM)?

Reply:
@gvancuts , It is here:
https://projectacrn.github.io/latest/tutorials/acrn_ootb.html#generate-a-user-vm-preempt-rt-image

Reply:
Thanks. I think it's be good to add a link to it (via a note or something). If @deb-intel is going to take another pass at this later, it can also be done at that time. Thanks!

Reply:
I have added note below the links about how to set up Service VM & preempt RT VM ootb image.